policyengine_us/parameters/gov/states/mn/tax/income/subtractions/social_security/reduction/start.yaml

```diff
@@ -10,11 +10,14 @@ metadata:
     href: https://www.revisor.mn.gov/statutes/cite/290.0132#stat.290.0132.26
```

Comment:
While you are here, could you move the uprating to the general metadata and use the `propagate_metadata_to_children` metadata tag